x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecuritySystemIndices.java

```diff
@@ -38,25 +38,33 @@
 
     public static final int INTERNAL_MAIN_INDEX_FORMAT = 6;
     private static final int INTERNAL_TOKENS_INDEX_FORMAT = 7;
+    private static final int INTERNAL_PROFILE_INDEX_FORMAT = 8;
+
+    public static final String INTERNAL_SECURITY_PROFILE_INDEX_8 = ".security-profile-8";
+    public static final String SECURITY_PROFILE_ALIAS = ".security-profile";
```

Comment:
I chose to not place these names alongside with other security index names in `RestrictedIndicesNames` class because `RestrictedIndicesNames` is likely obsolete since we have made all system indices also restricted. There are tests need to be fixed before we can remove the class. In the meantime, I don't think we need add anything more into it.

Reply:
SGTM.